src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,165 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.Calendar;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+public class ConsoleHandler {
+
+    private static final Map<String, String> commandInfo = new LinkedHashMap<>();
+
+    static {
+        commandInfo.put("create", "create [path] [file-name]\nCreate file by specified path.\n");
+        commandInfo.put("read",
+                "read [path] [file-name]\nRead a file by specified path to the terminal.\n");
+        commandInfo.put("info", "info [path] [file-name]\nGet short info about specified file.\n");
+        commandInfo.put("help", "help [command]\n"
+                + "Get info about all available commands or about a specified one.\n");
+        commandInfo.put("exit", "exit\nSay bye-bye :-)\n");
+    }
+
+    public void handleInput() throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String input = scanner.nextLine();
+            String[] split = input.split(" ");
+            switch (split[0]) {
+                case "create":
+                    create(split[1], split[2]);
```

Comment:
@Tarabarnik, can we create variables for each element of an array?
String operation = split[0];
String path = split[1];
...